pkg/elle/core/depend.go

```diff
@@ -32,6 +32,14 @@ const (
 	WRDepend DependType = "wr"
 	// RWDepend ...
 	RWDepend DependType = "rw"
+	// EXTKeyDepend ...
```

Comment:
emm...Should these be anomalies?

Reply:
A cycle would be an anomaly, the depend explains the steps in it only.

Reply:
`DependType` is for single depend, maybe anomaly should be other type.